src/common/containers/app.js

```diff
@@ -35,7 +19,7 @@ export class App extends Component {
         />
         <Header
           authenticated={this.props.auth.authenticated}
-          user={this.props.user.user}
+          user={this.props.user}
```

Comment:
`user` data is not nested in reducer anymore (as we don't need any `isFetching` etc).